mac

```diff
@@ -270,44 +303,62 @@ fancy_echo 'Checking on Ruby installation...'
 
 append_to_file "$HOME/.gemrc" 'gem: --no-document'
 
-if command -v rbenv >/dev/null || command -v rvm >/dev/null; then
-  fancy_echo 'We recommend chruby and ruby-install over RVM or rbenv'
-else
-  if ! brew_is_installed "chruby"; then
-    fancy_echo 'Installing chruby, ruby-install, and the latest Ruby...'
-
+case "$SHELL" in
+  */fish) :
+    fancy_echo 'Installing rbenv...'
     brew bundle --file=- <<EOF
-    brew 'chruby'
-    brew 'ruby-install'
+    brew 'rbenv'
 EOF
+    append_to_file "$shell_file" "status --is-interactive; and source (rbenv init -|psub)"
+    # rbenv currently doesn't have a convenience version to use, e.g., "latest",
+    # so we check for the latest version against Homebrew instead.
+    latest_ruby="$(brew info ruby | egrep -o "\d+\.\d+\.\d+" | head -1)"
+    if ! rbenv versions | ag "$latest_ruby" > /dev/null; then
+      rbenv install "$latest_ruby"
+      rbenv global "$latest_ruby"
+      eval "$(rbenv init - bash)"
+    fi
+    ;;
+  *) :
```

Comment:
This is good.  There was some discussion about rbenv being the default choice across the board.  After this one gets merged, I'll probably open a PR that does that and solicit feedback from #ruby.